session/pingpong/blockchain.go

```diff
@@ -61,3 +61,19 @@ func (bc *Blockchain) GetAccountantFee(accountantAddress common.Address) (uint16
 
 	return res.Value, err
 }
+
+// IsRegistered checks wether the given identity is registered or not
+func (bc *Blockchain) IsRegistered(registryAddress, addressToCheck common.Address) (bool, error) {
```

Comment:
why are we passing 'registryAddress'  here? We probably should construct it together with bc. Registry is not something that change?

Reply:
this component is just a caller for blockchain, it is up to the consumer to know which registry to call.